xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -914,6 +914,28 @@ distribution.automaticStartOnWiki=$xwikiPropertiesAutomaticStartOnWiki
 #-# By default this property is set to true:
 # url.trustedDomainsEnabled=true
 
+#-# [Since 15.0RC1]
+#-# [Since 14.10.4]
+#-# Define the list of schemes that are allowed for trusted URIs. Those schemes are checked whenever an absolute URI
+#-# needs to be checked (e.g. before performing a redirect). Any URI whose scheme doesn't belong to that list will not
+#-# be considered trustful, even if the domain of the URI is trusted.
+#-# Also note that even if a protocol is added here to be trusted, it might need a custom protocol handler. By default,
+#-# only http,https,ftp and file protocols are handled.
+#-#
+#-# The default is:
+# url.trustedSchemes=http,https,ftp
+
+#-# [Since 15.0RC1]
+#-# [Since 14.10.4]
+#-# Define the default scheme to fallback on in case an absolute URL is given starting with //
+#-# e.g.: //xwiki.org/myfile
+#-# Default value is https, but in some specific configuration admins might  want to switch it to http.
+#-# Note that for security reason, the value given here might be part of the trustedSchemes defined above, else it will
+#-# fallback to https.
+#-#
+#-# The default is:
+# url.defaultScheme=https
```

Comment:
>  However, I believe that in fact it absolutely doesn't matter for the code which scheme is chosen for URLs without scheme as the validation is only based on the host and the scheme is discarded.

You're completely right, I implemented this with the idea in mind that this would be used to actually perform a request, which is never the case in fact. So getting rid of it.